CONTRIBUTING.md

```diff
@@ -153,6 +153,23 @@ rather than conform to specific API names or argument patterns in the spec.
 
 For a deeper discussion, see: https://github.com/open-telemetry/opentelemetry-specification/issues/165
 
+## Running Tests Locally
+
+1. Go to your Contrib repo directory. `cd ~/git/opentelemetry-python-contrib`.
+2. Create a virtual env in your Contrib repo directory. `python3 -m venv my_test_venv`.
+3. Activate your virtual env. `source my_test_venv/bin/activate`.
+4. Clone the [OpenTelemetry Python](https://github.com/open-telemetry/opentelemetry-python) Python Core repo to a folder named `opentelemetry-python-core`. `git clone https://github.com/open-telemetry/opentelemetry-python.git opentelemetry-python-core`.
+5. Make sure you have `tox` installed. `pip install tox`.
+6. To sdist-package, install and test your project, just type `tox`
```

Comment:
I'd replace step 6. with something like, "Run `tox` without any arguments to run tests for all packages." and then link to tox documentation.